src/App.tsx

```diff
@@ -22,16 +22,16 @@ const useStyles = makeStyles(() => {
 function App() {
   const classes = useStyles()
   return (
-    <ThemeSwitcher storageKey="eth2.themeKey" themes={{ light: theme }}>
```

Comment:
I'm not sure if changing this is okay, but I couldn't find any reference/use of `eth2.themeKey`

Reply:
Yes, its ok, The themeswitcher uses this string to save current theme in localstorage.